supporter-product-data/build-tc

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+# build scala
+cd ..
+./sbt "project supporter-product-data" riffRaffUpload
```

Comment:
good idea making it a script.  FYI I recently found you can do `supporter-product-data / riffRaffUpload` to run a task in a project without having to switch to it